stdlib/src/hashlib.rs

```diff
@@ -45,17 +55,33 @@ mod hashlib {
         usedforsecurity: bool,
     }
 
+    impl From<NewHashArgs> for HashArgs {
+        fn from(args: NewHashArgs) -> Self {
+            Self {
+                string: args.data,
+                usedforsecurity: args.usedforsecurity,
+            }
+        }
+    }
+
+    #[derive(FromArgs)]
+    #[allow(unused)]
+    struct XofDigestArgs {
+        #[pyarg(positional)]
+        length: usize,
+    }
```

Comment:
```suggestion
        length: isize,
    }

    impl XofDigestArgs {
        fn length(&self, vm: &VirtualMachine) -> PyResult<usize> {
            if length < 0 {
                return vm.new_value_error("length must be non-negative".to_owned());
            }
            Ok(length as usize)
        }
    }
```
the skipped test may be fixed with this

Reply:
Ok let me test this later 👍